tdrs-backend/tdpservice/settings/cloudgov.py

```diff
@@ -158,7 +158,10 @@ class Staging(CloudGov):
     """Settings for applications deployed in the Cloud.gov staging space."""
 
     # TODO: why not just 'appcloudgov'?
```

Comment:
Can probably get rid of this TODO since we are specifically not using the `app.cloud.gov` in the TANF-Staging cf space anymore.